terraform/aws/analytical-platform/oidc/configuration/assumable-roles.json

```diff
@@ -176,12 +180,28 @@
   "modernisation-platform-lake-formation-share": {
     "account": "analytical-platform-data-production",
     "stateConfig": [],
-    "repositories": [
-      "ministryofjustice/modernisation-platform-environments"
-    ],
+    "repositories": ["ministryofjustice/modernisation-platform-environments"],
     "targets": [],
     "stateLockingDetails": [],
     "ssmParameterConfig": [],
     "lakeFormationSharePolicy": true
+  },
+  "create-a-derived-table-infrastructure": {
+    "account": "analytical-platform-data-production",
+    "stateConfig": [],
+    "repositories": ["ministryofjustice/create-a-derived-table-infrastructure"],
+    "targets": ["analytical-platform-data-production"],
+    "stateLockingDetails": [],
+    "ssmParameterConfig": []
+  },
+  "data-engineering-database-access": {
```

Comment:
I am pretty sure this at a minimum uses a state for Pulumi and now for terraform, so this will need amending.

Reply:
Both repositories are Pulumi Python and use the bucket `s3://data-engineering-pulumi.analytics.justice.gov.uk/.pulumi/`  and within the `stacks` folder are the files `de-database-access.json` and `create-a-derived-table-infra.json` which i have inferred are the state files for Pulumi. The stateConfig has been updated accordingly.   I have been unable to find any reference to a `terraform.tfstate`  in the `data-engineering-database-access` repository